### ISSUE TITLE
Import error after upgrading to 1.0.13 (HASS 0.111.1) on Beaglebone Black (armhf)

### ISSUE THREAD

**Issue by bhaonvashon** [claytonjn/hass-circadian_lighting#98]
After upgrading to 1.0.13, I see the following in the logs:

```
Jun 11 19:22:53 ha-hub hass[9721]: 2020-06-11 19:22:53 ERROR (MainThread) [homeassistant.setup] Error during setup of component circadian_lighting
Jun 11 19:22:53 ha-hub hass[9721]: Traceback (most recent call last):
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/numpy/core/__init__.py", line 24, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     from . import multiarray
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/numpy/core/multiarray.py", line 14, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     from . import overrides
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/numpy/core/overrides.py", line 7, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     from numpy.core._multiarray_umath import (
Jun 11 19:22:53 ha-hub hass[9721]: ImportError: libf77blas.so.3: cannot open shared object file: No such file or directory
Jun 11 19:22:53 ha-hub hass[9721]: During handling of the above exception, another exception occurred:
Jun 11 19:22:53 ha-hub hass[9721]: Traceback (most recent call last):
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/homeassistant/setup.py", line 190, in _async_setup_component
Jun 11 19:22:53 ha-hub hass[9721]:     result = await asyncio.wait_for(task, SLOW_SETUP_MAX_WAIT)
Jun 11 19:22:53 ha-hub hass[9721]:   File "/usr/lib/python3.7/asyncio/tasks.py", line 416, in wait_for
Jun 11 19:22:53 ha-hub hass[9721]:     return fut.result()
Jun 11 19:22:53 ha-hub hass[9721]:   File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Jun 11 19:22:53 ha-hub hass[9721]:     result = self.fn(*self.args, **self.kwargs)
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/config/custom_components/circadian_lighting/__init__.py", line 112, in setup
Jun 11 19:22:53 ha-hub hass[9721]:     interval, transition)
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/config/custom_components/circadian_lighting/__init__.py", line 138, in __init__
Jun 11 19:22:53 ha-hub hass[9721]:     self.data['timezone'] = self.get_timezone()
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/config/custom_components/circadian_lighting/__init__.py", line 157, in get_timezone
Jun 11 19:22:53 ha-hub hass[9721]:     from timezonefinder import TimezoneFinder
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/timezonefinder/__init__.py", line 2, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     from .timezonefinder import TimezoneFinder
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/timezonefinder/timezonefinder.py", line 8, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     import numpy as np
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/numpy/__init__.py", line 142, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     from . import core
Jun 11 19:22:53 ha-hub hass[9721]:   File "/home/debian/.installs/hass/venv/lib/python3.7/site-packages/numpy/core/__init__.py", line 50, in <module>
Jun 11 19:22:53 ha-hub hass[9721]:     raise ImportError(msg)
Jun 11 19:22:53 ha-hub hass[9721]: ImportError:
Jun 11 19:22:53 ha-hub hass[9721]: IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!
Jun 11 19:22:53 ha-hub hass[9721]: Importing the numpy C-extensions failed. This error can happen for
Jun 11 19:22:53 ha-hub hass[9721]: many reasons, often due to issues with your setup or how NumPy was
Jun 11 19:22:53 ha-hub hass[9721]: installed.
Jun 11 19:22:53 ha-hub hass[9721]: We have compiled some common reasons and troubleshooting tips at:
Jun 11 19:22:53 ha-hub hass[9721]:     https://numpy.org/devdocs/user/troubleshooting-importerror.html
Jun 11 19:22:53 ha-hub hass[9721]: Please note and check the following:
Jun 11 19:22:53 ha-hub hass[9721]:   * The Python version is: Python3.7 from "/home/debian/.installs/hass/venv/bin/python3.7"
Jun 11 19:22:53 ha-hub hass[9721]:   * The NumPy version is: "1.18.5"
Jun 11 19:22:53 ha-hub hass[9721]: and make sure that they are the versions you expect.
Jun 11 19:22:53 ha-hub hass[9721]: Please carefully study the documentation linked above for further help.
Jun 11 19:22:53 ha-hub hass[9721]: Original error was: libf77blas.so.3: cannot open shared object file: No such file or directory
Jun 11 19:22:53 ha-hub hass[9721]: 
Jun 11 19:22:54 ha-hub hass[9721]: 2020-06-11 19:22:54 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform circadian_lighting.switch: Unable to set up component.
Jun 11 19:22:54 ha-hub hass[9721]: 2020-06-11 19:22:54 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform circadian_lighting.switch: Unable to set up component.
Jun 11 19:22:54 ha-hub hass[9721]: 2020-06-11 19:22:54 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform circadian_lighting.switch: Unable to set up component.
Jun 11 19:22:54 ha-hub hass[9721]: 2020-06-11 19:22:54 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform circadian_lighting.switch: Unable to set up component.
Jun 11 19:23:07 ha-hub hass[9721]: 2020-06-11 19:23:07 ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform circadian_lighting.sensor: Unable to set up component.
```
Looking at [Troubleshooting ImportError](https://numpy.org/devdocs/user/troubleshooting-importerror.html) per the instructions in the log, indeed `libatlas-base-dev` is not installed:

```
[debian@ha-hub ha]$ apt-cache policy libatlas-base-dev
libatlas-base-dev:
  Installed: (none)
  Candidate: 3.10.3-8
  Version table:
     3.10.3-10 -10
        -10 http://deb.debian.org/debian testing/main armhf Packages
     3.10.3-8 900
        900 http://deb.debian.org/debian buster/main armhf Packages
```
Do I need to install this library manually or should this be part of the installation setup of Circadian Lighting?

I am running HASS on a Beaglebone Black running an [official Debian image](https://beagleboard.org/latest-images) (not the latest).

```
[debian@ha-hub ha]$ arch
armv7l
[debian@ha-hub ha]$ cat /etc/dogtag
BeagleBoard.org Debian Image 2019-12-16
[debian@ha-hub ha]$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```

**Comment by bhaonvashon** [comment]
Oh, BTW, I'm running HASS in a venv.

**Comment by claytonjn** [comment]
I had a feeling this would cause issues. I kept external imports out until now intentionally so that there wouldn't be any issues installing dependencies. 

CL only relies on timezonfinder, which is included as a requirement in manifest.json and your system did attempts to install that. It looks like timezonfinder timezonefinder requires numpy which also attempted to install but then hit the libatlas-base-dev issue. It seems like this is a somewhat isolated issue (1.0.12b included the timezonfinder requirement and was released back in mid-March) so I would try manually installing that library. I don't think it should be added to the requirements of CL because I'm assuming it's system dependent.
